### PR TITLE
Fix Mayavi visualizer.

### PR DESCRIPTION
**Pull request by pkreissl**
The visualizer was not able to get the particle type. Fixed.

### CODE REVIEW

File: doc/tutorials/python/08-visualization/scripts/simulation.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from __future__ import print_function
-import espressomd._system as es
+
```
Comment (KaiSzuttor) [inline doc/tutorials/python/08-visualization/scripts/simulation.py:2]: Can you please just remove that empty lines?

Comment (pkreissl) [reply doc/tutorials/python/08-visualization/scripts/simulation.py:2]: I usually leave a blank line after `__future__ `imports, as they are required to appear before any code (except docstrings) and in contrast to other imports are not script-specific -- just a part of the preamble.
I think this is also the way pep8 suggests (see https://www.python.org/dev/peps/pep-0008/#imports section 'Module level dunder names').

**Review by fweik — APPROVED** [review]

File: doc/tutorials/python/08-visualization/scripts/simulation.py (hunk 1 above)
Comment (fweik) [reply doc/tutorials/python/08-visualization/scripts/simulation.py:2]: We have bigger fish to fry...

**Comment by RudolfWeeber** [comment]
It seems to me like a matter of taste and therefore maybe beyoud the scope of QA.

File: doc/tutorials/python/08-visualization/scripts/simulation.py (hunk 1 above)
Comment (KaiSzuttor) [reply doc/tutorials/python/08-visualization/scripts/simulation.py:2]: Thats why I just use autopep8 -i

**Pull request merged**